lib/generator/swaggerMocker.ts

```diff
@@ -92,7 +92,7 @@ export default class SwaggerMocker {
       return;
     }
     return {
-      [headerAttr]: "LocationURl",
+      [headerAttr]: "https://foo.com/operationstatus",
```

Comment:
Why change to this ?

Reply:
autorest validates the value of these two headers which have to be a URL.